### PR TITLE
Fixed character styling on multiplayer page

### PR DESCRIPTION
**Pull request by EsdrasBrutus**
# Description

Trello: https://trello.com/c/Td3rBj3u/727-multiplayer-character-sizing-styling-fix

What I did:
I added `@media` breakpoints for mobile screens and adjusted the `margin`s, and image sizes of the characters on the multiplayer page. To overall make the page look cleaner. 

Before and After Changes:

before and after2
character-styling

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Change Status

- [x] Complete, tested, ready to review and merge

## Has This Been Tested

- [x] Yes

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

https://user-images.githubusercontent.com/10622915/146848574-779b1460-e2cf-4bc3-815f-821eef7bad8a.mp4

### CODE REVIEW

**Review by Fry-Fr — COMMENTED** [review]
Good find and fixing this.

File: src/styles/less/NextSteps.less [hunk 1]
```diff
@@ -86,13 +86,41 @@
...
+        left: -15%;
+        top: 0;
+    }
+
+    @media (max-width: 380px) {
+        transform: scale(80%);
+        left: -5rem;
+        top: 0;
+    }
 }
```
Comment (felipe-sq) [inline src/styles/less/NextSteps.less:104]: Great job including the additional breakpoints for smaller screens. The before and after images are very helpful to visualize the differences for anyone who cannot readily test the code on their local machine!

**Review by felipe-sq — COMMENTED** [review]
The PR description and checklists are detailed and helpful! I would add a quick loom video with a walkthrough of your changes and maybe a bit about the thought process behind it. Otherwise, this looks great!

**Review by mel147 — APPROVED** [review]
Great job Esdras! Yes, as Felipe said, it's a great idea to incorporate a Loom video of the changes you made. Other than that, nice fix and good job looking at the details closely to find the problem in the first place.

**Review by dujules23 — APPROVED** [review]
This looks good! Way to go on making sure this part of the app maintains it's consistency in regard to accessibility. Thanks for the Loom video as well which provided a good description of what was completed.

**Comment by felipe-sq** [comment]
Great job with the loom video! Thank you for interacting with and responding to the PR comments, too.

**Review by felipe-sq — APPROVED** [review]

**Review by StnsGeneral — APPROVED** [review]
Nice catch on this. Also quite like the use of responsive units. My concern before I saw that was that perhaps the font would have gotten too small to read on some screen sizes. Beautiful work.

**Pull request merged**